README.md

```diff
@@ -1,7 +1,6 @@
-<h1 align="center">
-	<img width="594" src="https://cdn.rawgit.com/derimagia/awesome-alfred-workflows/master/media/header.gif" alt="awesome-alfred-workflows">
-	<br>
-</h1>
+# Awesome Alfred Workflows
+
+![Awesome Alfred Workflows](preview.gif "Awesome Alfred Workflows")
```

Comment:
This isn't centered, although you may be able to remove the <img> tag for this. I personally like the old image better (without the wallpaper clipped in)

Reply:
I went ahead and reverted this so that it's centered and also tried to recreate the gif without the background (better cropped) in my next commit